### ISSUE TITLE
Do i need to add parts 

### ISSUE THREAD

**Issue by Player22120** [JuniorDjjr/VehFuncs#43]
Do i need to add parts to a car like windscreen wiper to i need to model like a wiper or not. Pls answer cause i want to make gta sa car work in vehfunchs.

**Comment by Player22120** [comment]
btw im using zmodeler 2.2

**Comment by Angelomon** [comment]
Yes you need to add to the car a model of windscreen wiper; then put the function for [Windshield wipers](https://github.com/JuniorDjjr/VehFuncs/wiki/%5BEN%5D-Windshield-wipers) so they work when it rains.
